src/test/java/hudson/plugins/git/GitSCMTest.java

```diff
@@ -480,7 +480,7 @@ public void testExcludedRegionMultiCommit() throws Exception {
         assertTrue("scm polling did not detect changes in server project", serverProject.poll(listener).hasChanges());
     }
 
-    /**
+    /*
```

Comment:
Removed javadoc comment.  There’s a lot of places where you removed the javadoc comment and I'm not sure why.

Reply:
Hi @samrocketman, thanks for asking. A brief summary of changes was already given in the pull request description. Let me provide some more details here.

The origin of this pull request was running `mvn javadoc:test-javadoc` which reported a lot of errors. So I followed along and fixed all of these.

Having a comment on a test method may be fine if there are some details to describe that cannot be codified.

Having a Javadoc on a test method (not test helper method or class) is questionable because Javadoc is for API documentation and test methods are not a software's API. So the use on test methods is not correct. Therefore I converted Javadoc on test methods to comments. Maybe I should have converted even more.

A lot of Javadocs seemed autogenerated, just mentioning the tested method. This information is not very useful as it should better be stated in the test method name and a clear implementation. Addionally it was not valid Javadoc most of the time because some rules haven't been followed. These Javadocs have been removed.

To write valid Javadoc a bunch of rules have to be followed - e.g. describing all parameters, exceptions, return value and having proper formatting. This wasn't the case for many Javadocs on the test code. If the Javadoc use was correct (e.g. on test infrastructure code) but the Javadoc itself was invalid it has been corrected.

Reply:
> Having a Javadoc on a test method (not test helper method or class) is questionable because Javadoc is for API documentation and test methods are not a software's API.

Good point.  I like the use of `@Issue` annotations as well.